content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md

```diff
@@ -12,28 +12,35 @@ When one or more of these resources reach specific consumption levels, the
 kubelet can proactively fail one or more pods on the node to reclaim resources
 and prevent starvation.
 
-During a node-pressure eviction, the kubelet sets the `PodPhase` for the
-selected pods to `Failed`. This terminates the pods.
+During a node-pressure eviction, the kubelet sets the [phase](/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase) for the
+selected pods to `Failed`, and terminates the Pod.
 
 Node-pressure eviction is not the same as
 [API-initiated eviction](/docs/concepts/scheduling-eviction/api-eviction/).
 
-The kubelet does not respect your configured `PodDisruptionBudget` or the pod's
+The kubelet does not respect your configured {{<glossary_tooltip term_id="pod-disruption-budget" text="PodDisruptionBudget">}}
+or the pod's
 `terminationGracePeriodSeconds`. If you use [soft eviction thresholds](#soft-eviction-thresholds),
 the kubelet respects your configured `eviction-max-pod-grace-period`. If you use
-[hard eviction thresholds](#hard-eviction-thresholds), it uses a `0s` grace period for termination.
+[hard eviction thresholds](#hard-eviction-thresholds), the kubelet uses a `0s` grace period (immediate shutdown) for termination.
+
+## Self healing behavior
+
+The kubelet attempts to [reclaim node-level resources](#reclaim-node-resources)
+before it terminates end-user pods. For example, it removes unused container
+images when disk resources are starved.
 
 If the pods are managed by a {{< glossary_tooltip text="workload" term_id="workload" >}}
 resource (such as {{< glossary_tooltip text="StatefulSet" term_id="statefulset" >}}
 or {{< glossary_tooltip text="Deployment" term_id="deployment" >}}) that
-replaces failed pods, the control plane or `kube-controller-manager` creates new
+replaces failed pods, the control plane (`kube-controller-manager`) creates new
 pods in place of the evicted pods.
 
-{{<note>}}
-The kubelet attempts to [reclaim node-level resources](#reclaim-node-resources)
-before it terminates end-user pods. For example, it removes unused container
-images when disk resources are starved.
-{{</note>}}
+If you are running a [static pod](/docs/concepts/workloads/pods/#static-pods)
+and the kubelet is under resource pressure, the kubelet may evict that static
+Pod. The kubelet then tries to create a replacement.
```

Comment:
"and the kubelet is ..., the kubelet may ... The kubelet then ..."

Do we need all these "kubelet" in this sentence?

A static pod **may** be evicted ... this is confusing. How do I know if my static Pod
will be evicted? The kubelet then tries to create a replacement ...
The replacement has to run on the same node, right?
How come this behavior is called an eviction? The kubelet "evicts" a static Pod
only for the purpose of recreation?

It is one thing whether the design/implementation makes sense; it is another thing
whether we are clarifying some concepts or behaviors rather than making it more
confusing.

Reply:
I reworded; does this help?